test/e2e/helpers_test.go

```
@@ -56,22 +34,22 @@ func createRuleViaAPI(t *testing.T, f *framework.Framework, ctx context.Context,
 
 	resp, err := f.HTTPClient().Do(req)
 	if err != nil {
-		t.Fatalf("Failed to make create request for %s: %v", alertName, err)
+		return "", fmt.Errorf("make create request: %w", err)
 	}
 	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)